HISTORY.md

```diff
@@ -0,0 +1,259 @@
+with only small differences in actual usage,
+and that all three would benefit everyone.
+Discussion is sparked in the Gist’s comments.
+
+Galvanized by the Gist,
+[@js-choi][] switches from writing the [smart-mix-pipes spec][]
+to writing a [Hack-pipes spec][].
+
+## 2021-07
+[@tabatkins][] (Google) schedules a
+[TC39 incubator meeting devoted to pipes][2021-07 incubator]
+(see [general information on incubator calls][]).
+Attendees of special note in this meeting are [@syg][] (Google V8),
+[@littledan][] (Igalia), [@rbuckton][] (Microsoft), and [@ljharb][]. [@codehag][] (Mozilla SpiderMonkey) is unable to attend.
+
+[@tabatkins][] (Google) presents three choices to the attendees again:
+F# pipes, Hack pipes, and Elixir pipes.
+
+[@rbuckton][] (Microsoft) is still in favor of F# pipes with his proposed syntax for
+[partial function application][] (see [§ 2016–2017](#20162017)).
+[@rbuckton][] (Microsoft) debates with [@tabatkins][] (Google)
+about whether F# pipes or Hack pipes are more intuitive.
+(He also mentions that [@codehag][] (Mozilla SpiderMonkey)
+might be interested in co-championing partial function application without pipes,
+based on her user studies’ findings; see [§ 2018–2020](#20182020).)
+
+[@syg][] (Google V8) voices concerns again about engine performance
+of partial function application and F# pipes (see [§ 2018–2020](#20182020)).
+
+[@tabatkins][] (Google) and [@ljharb][] are supportive
+of either Hack pipes or F# pipes:
+“90%” F# pipes and “100%” Hack pipes.
+However, their 90% support of F# pipes would drop a lot if [@syg][]
+(Google V8)’s performance concerns about F# pipes are borne out.
+
+Nobody in the meeting seems to think Elixir pipes are a good fit for JavaScript.
+
+Most everyone in the meeting seems to be in favor of picking some style for pipes
+after three years of indecision.
+
+## 2021-08
+[@tabatkins][] (Google) plans to present *some* pipe-operator style for Stage 2.
+Based on the results of the the preceding meeting, they pick Hack pipes.
+This has support from [@ljharb][] and some other TC39 members.
+
+Through this month, [@tabatkins][] (Google) continues to debate offline
+with [@mAAdhaTTah][] regarding Hack pipes vs. F# pipes.
+As a result, [@mAAdhaTTah][] changes his mind from being in favor of F# pipes
+to being in favor of Hack pipes,
+deciding that Hack pipes would be better for bridging functional programming
+with the rest of the JavaScript ecosystem.
+
+[@rbuckton][] (Microsoft) joins in debating with [@tabatkins][] (Google) in late August.
+[@rbuckton][] (Microsoft) notes the groundswell of support among TC39 about Hack pipes
+due to “some of the limitations of F# pipes’ limitations”.
+Therefore feeling that F# pipes would continue to be indefinitely stuck at an impasse,
+[@rbuckton][] (Microsoft) thus decides to give “tentative agreement” to Hack pipes.
+(See [@rbuckton’s narrative][].)
+
+On 2021-08-31, [@tabatkins][] (Google) therefore
```

Comment:
I believe this date should be treated as a title for coherence with the rest

Reply:
Thanks for the comment!

There aren’t any other headings for specific days, so making a “2021-08-31” would be inconsistent with them. I also think the part right before this is essential lead-up to this part. 

Do you think that separating this part with a `***`/`<hr>` would be okay?